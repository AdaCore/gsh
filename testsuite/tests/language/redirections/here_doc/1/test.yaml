title: Test basic Here-Document construction
