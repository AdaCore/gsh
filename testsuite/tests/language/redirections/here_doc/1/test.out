Hi I am a here document

Seems to work
